deployments.json

```diff
@@ -89,7 +89,8 @@
           "height": 4139174
         }
       }
-    }
+    },
+    "channels": [1]
```

Comment:
Do these only make sense in the context of hubble?

Reply:
somewhat, the frontend will query hubble to populate the deployments doc